demos/interactive/scroll-grid-container.php

```diff
@@ -28,7 +28,7 @@
 $g1->addActionButton('red', function ($js) {
     return $js->closest('tr')->css('color', 'red');
 });
-// THIS SHOULD GO AFTER YOU CALL addAction() !!!
+// THIS SHOULD GO AFTER YOU CALL Grid::addActionButton() !!!
```

Comment:
@DarkSide666 why? added in https://github.com/atk4/ui/pull/635/files#diff-77da0ad02520ab833baf119292167fc528e59c732fcf530278535ce4a101c426R26

also, when the demo is scrolled, the newly loaded rows does not have the buttons:

![image](https://user-images.githubusercontent.com/2228672/178722787-8cac6019-dc4b-4b65-b2c7-5d8e812bd6b9.png)

also the width is broken - an SUI issue?